src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -222,13 +219,10 @@ export default defineComponent({
       return 'total';
     });
 
-    const totalAmount = computed(() => getTxAmountTotal.value(
-      txWrapped.value,
-      getTxDirection.value(props.transaction),
-    ));
+    const totalAmount = computed(() => getTxAmountTotal.value(txWrapped.value, direction.value));
```

Comment:
This change makes the code less readable in my opinion. Less lines != better code :)